### ISSUE TITLE
WSL setup: Current instructions mean BROWSER variable will be removed from zshrc

### ISSUE THREAD

**Issue by Disco-Chef** [lewagon/setup#303]
Hi again 😅 

With the current order of the setup, we set the BROWSER variable in the `zshrc` just before authorizing the `gh` cli, seeing as the terminal will try to open a browser to authenticate.

However, when they will fork and clone the dotfiles and run the `install.sh`  script, that BROWSER variable in the  `zshrc` is gone.

I propose we adapt the `install.sh` script to keep that line in the final `zshrc`

🖖 🙇

**Comment by dmilon** [comment]
Hi @Disco-Chef :wave: 

I agree, there is definitely something to improve about how we define the BROWSER environment variable and how we keep it after updating the `.zshrc` file. I'm going to have a look into it ASAP. 

This is related to https://github.com/lewagon/setup/issues/305

**Comment by Disco-Chef** [comment]
Had an interesting issue today that might make this issue obsolete:

A data student (wsl) was wondering why `jupyter notebook` would not open the notebook on their browser automatically, like their buddies (also on wsl).

Turns out, that particular data student had the `BROWSER` variable defined in his zshrc (even though with the current setup order, that would have been overwritten i think once they `zsh install.sh` in their dotfiles) while his buddies did not have the `BROWSER` variable set in their zshrc.

Sure enough, once we removed the `export BROWSER` line from his zshrc, `jupyter notebook` opened up the notebook automatically in the browser, just like his buddies 😄 

I have reproduced this on my windows machines too. 

I also ran the `gh` cli commands that should open up a tab in the browser (like `gh auth login -s 'user:email' -w` and `gh repo view --web` on my windows machines without the `BROWSER` variable defined and sure enough they launch a tab on the browser just fine without it 👍 

So it would seem that defining the `BROWSER` variable before the dotfiles steps (which lead to it being overwritten/removed when running `zsh install.sh` symlinking, is OK for both the web and data students 🙌 

Will do a bit more testing and see if this is the case for the web people using browsers other than Chrome (which is what I was using on my machines) 🖖
